gen.sh

```diff
@@ -1,67 +1,31 @@
 #!/bin/bash
-# Script to generate the Protobuf classes from googleapis/google-cloudevents
 set -e
 
-echo "~ START"
-PROTOBUF_VERSION=3.12.3
+EVENTS_SPEC_REPO='https://github.com/michaelawyu/google-cloudevents'
```

Comment:
No personal projects please.

Reply:
As stated in this PR, the generation of the Python library makes a few assumptions about the JSON schemas, which are not incorporated in the main repo yet. Switch to the main repo here will fail the generation process.

Reply:
Yup, I understand there are extra changes in the fork.

Can we first add the JSON schemas to the official repo before creating this PR?
https://github.com/googleapis/google-cloudevents/compare/master...michaelawyu:master
I don't see a PR to add these schemas. That seems like a large dependency for this solution.

I don't imagine how we can merge this PR without having the schemas in the main repo first.